xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-test/xwiki-platform-livedata-test-pageobjects/src/main/java/org/xwiki/livedata/test/po/TableLayoutElement.java

```diff
@@ -489,6 +491,16 @@ public void filterColumn(String columnLabel, String content, boolean wait, Map<S
         } else if (classes.contains("filter-text")) {
             element.clear();
             element.sendKeys(content);
+        } else if (classes.contains("filter-date")) {
+            element.click();
+            DateRangePicker picker = new DateRangePicker(element);
+            if (StringUtils.isNotBlank(content)) {
+                element.clear();
+                element.sendKeys(content);
+                picker.applyRange();
```

Comment:
This feels a bit strange. You're not reproducing the user behavior. I doubt users are typing in the date range text input. I assume they use the Data Range Picker UI to select the start and end date most of the time. Isn't this what we want to test?